openedx/adg/lms/static/components/student_account/PasswordResetInput.jsx

```diff
@@ -0,0 +1,34 @@
+// Changes from Core
+// 1. Added span after InputText.
+// 2. Updated type of input text.
+
+/* globals gettext */
+
+import PropTypes from 'prop-types';
+import React from 'react';
+
+import { InputText } from '@edx/paragon/static';
+
+function PasswordResetInput(props) {
+  return (
+    <div className="form-field">
+      <InputText
+        id={props.name}
+        type={props.type}
+        themes={['danger']}
+        dangerIconDescription={gettext('Error: ')}
+        required
+        {...props}
+      />
+      <span className='show-pass-icon toggle-password password-change-eye-icon' onClick={props.onClickHandler}>
+          <img src="https://static-omnipreneurshipacademy.s3.eu-central-1.amazonaws.com/icons/eye-icon.png"/>
```

Comment:
Can we avoid this hardcoded static url?

Reply:
any suggestions? If we want to send in props that will take some core changes. Also This is how it is done by the designer previous for login and register page and we will have to do core changes for that as well.